app/src/main/java/ingsw/group1/findmyphone/alarm/AlarmMessageParser.java

```diff
@@ -0,0 +1,33 @@
+package ingsw.group1.findmyphone.alarm;
+
+/**
+ * Parser class of static methods that provides composition of message for alarm request and response
+ * Message request is composed as:
+ * {@link AlarmMessageParser#ALARM_MESSAGE_REQUEST}
+ *
+ * @author Giorgia Bortoletti
+ */
+public class AlarmMessageParser {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'AlarmMessageParser' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929934)